### DISCUSSION TITLE
Login Error Connection Refused after Windows to Linux Switch (Dual Boot)

### DISCUSSION THREAD

**Discussion by jaketuring999** [rustdesk/rustdesk#6960] [General]
### Bug Description

I have dual boot system with Linux Mint Ubuntu and Windows.
I run this bash script on my Linux to get to Windows
`
read -p "Are you sure you want to switch to Windows? (y/n): " -n 1 -r
echo    # Move to a new line

if [[ $REPLY =~ ^[Yy]$ ]]
then
    sudo efibootmgr -n 0000
    sudo efibootmgr -o 0002,0000
    echo "Rebooting to Windows..."
    sleep 3  # Optional: gives a short pause before rebooting
    sudo reboot
else
    echo "Operation cancelled."
fi

`
When on my windows system I'm able to remote in with rust desk and works fine, Then I restart my system to return to Linux and receive the error
 "Login Error connection Refused". If I go to the computer manually and type my password in then I'm able to connect to Linux System again.

### How to Reproduce

1. Have dual boot PC with rust desk on Linux and Windows
2. Local computer can connect to both
3. Reboot Linux into Windows with script (or your own version of it) 
4. Reboot Windows into Linux
5. Try to Login with Local machine = Connection refused 
6. Login manually to remote machine (type in password, click enter) then Connection accepted and works as expected

### Expected Behavior

After reboot from windows to linux, I should be able to login to the Linux machine without manually typing in password first. 

If I boot normally, from shutoff to linux, I can connect as expected without having to manually type in password. 

### Operating system(s) on local side and remote side

macOS Monterey 12.4 -> Linux Mint 20.2 Cinnamon

### RustDesk Version(s) on local side and remote side

1.2.3 -> 1.2.3

### Screenshots

<img width="1319" alt="Screen Shot 2024-01-22 at 4 42 46 PM" src="https://github.com/rustdesk/rustdesk/assets/110423575/6ef5c174-c90b-44df-97cf-de9e1ef8465b">

https://github.com/rustdesk/rustdesk/assets/110423575/d166d872-a3f8-4a94-802f-bc8a7ca7b507

**Comment by roborew** [comment]
An old post, but came up in google when I had a similar problem after rebooting from updates.

A simple workaround for this is turn on auto login on boot for the user. Make sure you have ssh access to the remote machine (many ways to achieve this, twingate etc). 
login and run:
`sudo nano /etc/gdm3/custom.conf`
Confirm that wayland is false:
`WaylandEnable=false`
Then also uncomment:
`AutomaticLoginEnable = true`
`AutomaticLogin = YourUsername`
then
`sudo reboot`

The machine will now login into the user account without a password allowing you to connect with RustDesktop. However, this does now present a security risk. Recommended to now comment those lines again. Then lock the screen when you finish the session (turn on auto lock). These steps will need to be repeated every time you reboot. Other approaches might also work, updating the service file etc.
